playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.Functions/MyAzureBlobTrigger.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace AzureFunctionsEndToEnd.Functions;
 
-public class MyAzureBlobTrigger(ILogger<MyAzureBlobTrigger> logger, BlobContainerClient containerClient)
+public class MyAzureBlobTrigger(BlobContainerClient containerClient, ILogger<MyAzureBlobTrigger> logger)
 {
     [Function(nameof(MyAzureBlobTrigger))]
-    [BlobOutput("test-files/{name}.txt", Connection = "blob")]
-    public async Task<string> RunAsync([BlobTrigger("blobs/{name}", Connection = "blob")] string triggerString, FunctionContext context)
+    [BlobOutput("test-files/{name}.txt", Connection = "blob-blobs")]
```

Comment:
This connection name seems wrong. Nowhere did we say `blob-blobs` in the AppHost.